codepropertygraph/src/main/scala/io/shiftleft/passes/CpgPass.scala

```diff
@@ -66,6 +67,7 @@ abstract class ForkJoinParallelCpgPass[T <: AnyRef](
     prefix: String = ""
   ): Unit = {
     baseLogger.info(s"Start of pass: $name")
+    TimeMetric.initiateNewStage(getClass.getSimpleName, Some(name), "ForkJoinParallelCpgPass")
```

Comment:
Would something like the following achieve the same?
```suggestion
    TimeMetric.initiateNewStage(getClass.getSimpleName, Some(name), getClass.getName)
```

Reply:
+1 - a constant `ForkJoinParallelCpgPass` is wrong and misleading here, no?

Reply:
`getClass.getName` will return the implementation class name. In order to understand if the given pass is extending "ForkJoinParallelCpgPass" or  "ConcurrentWriterCpgPass" . But we can read the parent class names and incorporate that as well. I will make that change.

Reply:
BTW I am already passing  `Some(name)` => `Some(getClass.getName)`

Reply:
Done